onelogin/client.go

```diff
@@ -91,6 +88,11 @@ type GetUserByEmailResponse struct {
 	}
 }
 
+type Device struct {
+	DeviceId   int    `json:"device_id"`
```

Comment:
The Golang style prefers `DeviceID` over `DeviceId`. I'm pretty sure there are more such cases in the codebase but let's fix this one first since we are already here :-)